### ISSUE TITLE
Error when installing

### ISSUE THREAD

**Issue by NGOtaku** [kivy/kivy#5699]
i was doing the second part of the installation (python -m pip install docutils pygments pypiwin32 kivy.deps.sdl2 kivy.deps.glew) on cmd. i have python 3.4.4. i always get this code:

Could not find a version that satisfies the requirement pywin32>=223 (from pypiwin32) (from versions: )
No matching distribution found for pywin32>=223 (from pypiwin32)

what do i do?
yes, i confirmed the version by typing python --version

**Comment by bionoid** [comment]
I ran into this a while back too, iirc pywin32 has dropped support for Python 3.4. Probably the easiest solution is to upgrade Python to 3.5 or 3.6

**Comment by NGOtaku** [comment]
I first installed python 3.6.5 but for some reason, it wasnt recognized by cmd so i concluded it wasnt installed correctly. Then i reinstalled many times but there was no change. 
I tried to get 3.5 but there was no actual installer, just this zip with no exe (i dont know how to install it)
Thus i settled with 3.4

**Comment by bionoid** [comment]
> I first installed python 3.6.5 but for some reason, it wasnt recognized by cmd so i concluded it wasnt installed correctly.

I don't think it normally adds itself to PATH.. pretty sure 3.4 didn't do that for me either. I suggest you uninstall Python 3.4 (to avoid confusion), then install 3.6.5. Once it's installed, in cmd:

    setx path "c:\python365;c:\python365\scripts;"

Replace `c:\python365` with the actual directory where you installed Python

**Comment by tshirtman** [comment]
I
